modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionOverview/SubscriptionOverview.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import classNames from 'classnames';
+import {useEffect, useState} from 'react';
+import {useOutletContext} from 'react-router-dom';
+import i18n from '../../../../../../../common/I18n';
+import {useAppPropertiesContext} from '../../../../../../../common/contexts/AppPropertiesContext';
+import {getAccountSubscriptions} from '../../../../../../../common/services/liferay/graphql/queries';
+import {useCustomerPortal} from '../../../../../context';
+import {actionTypes} from '../../../../../context/reducer';
+import {
+	PRODUCT_TYPES,
+	SUBSCRIPTIONS_STATUS,
+} from '../../../../../utils/constants';
+import {getWebContents} from '../../../../../utils/getWebContents';
+import CardSubscription from '../SubscriptionOverview/components/CardSubscription';
+import SubscriptionsFilterByStatus from '../SubscriptionOverview/components/SubscriptionsFilterByStatus';
+import SubscriptionsNavbar from '../SubscriptionOverview/components/SubscriptionsNavbar';
+import '../../app.scss';
+
+const SubscriptionOverview = () => {
```

Comment:
Since is on Subscriptions, I believe we should name the component to SubscriptionsOverview